spec/controllers/schedule_rules_controller_spec.rb

```diff
@@ -1,178 +1,162 @@
-    expect(@price_policy).to be_valid
-    @params = { facility_id: @authable.url_name, instrument_id: @instrument.url_name }
-  end
-
-  context "index" do
+  describe "index" do
+    let(:staff) { create(:user, :staff, facility: facility) }
 
-    before :each do
-      @method = :get
-      @action = :index
+    before do
+      sign_in staff
+      get :index, product_params
     end
 
-    it_should_allow_operators_only do |_user|
-      expect(assigns[:instrument]).to eq(@instrument)
+    it "sets the product and renders" do
+      expect(assigns(:product)).to eq(product)
       expect(response).to be_success
       expect(response).to render_template("schedule_rules/index")
     end
-
   end
 
-  context "new" do
-
-    before :each do
-      @method = :get
-      @action = :new
+  describe "new" do
+    before do
+      sign_in senior_staff
+      get :new, product_params
     end
 
-    it_should_allow_managers_and_senior_staff_only do
-      expect(assigns[:instrument]).to eq(@instrument)
+    it "sets the product and renders" do
+      expect(assigns(:product)).to eq(product)
       expect(response).to be_success
       expect(response).to render_template("schedule_rules/new")
     end
 
+    it "sets the defaults to 9-5" do
+      expect(assigns(:schedule_rule).start_hour).to eq(9)
+      expect(assigns(:schedule_rule).start_min).to eq(0)
+      expect(assigns(:schedule_rule).end_hour).to eq(17)
+      expect(assigns(:schedule_rule).end_min).to eq(0)
+    end
   end
 
-  context "create" do
-
-    before :each do
-      @method = :post
-      @action = :create
-      @params.merge!(
-        schedule_rule: FactoryGirl.attributes_for(:schedule_rule, instrument_id: @instrument.id),
-      )
+  describe "create" do
+    def do_request
+      sign_in senior_staff
+      post :create, product_params.merge(schedule_rule: rule_params)
     end
 
-    it_should_allow_managers_and_senior_staff_only :redirect do
-      expect(assigns(:schedule_rule)).to be_kind_of ScheduleRule
-      is_expected.to set_flash
-      assert_redirected_to facility_instrument_schedule_rules_url(@authable, @instrument)
+    let(:rule_params) { FactoryGirl.attributes_for(:schedule_rule, instrument_id: product.id) }
+
+    it "creates the schedule rule and redirects to index" do
+      expect { do_request }.to change(ScheduleRule, :count).by(1)
+      expect(response).to redirect_to facility_instrument_schedule_rules_url(facility, product)
     end
 
     context "with restriction levels" do
-      before :each do
-        @restriction_levels = []
-        3.times do
-          @restriction_levels << FactoryGirl.create(:product_access_group, product_id: @instrument.id)
-        end
-        sign_in(@admin)
-      end
+      let!(:restriction_levels) { FactoryGirl.create_list(:product_access_group, 3, product_id: product.id) }
```

Comment:
Why are we calling these restriction levels? Can we use a name that is more in line with our terminology elsewhere in the app?